src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleFiltering/style.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from "@emotion/styled";
+import {
+  Dropdown,
+  DropdownPopper,
+  fontHeaderXs,
+  getColors,
+  getSpaces,
+  InputDropdown,
+} from "czifui";
+import React from "react";
+import { NewTabLink } from "src/common/components/library/NewTabLink";
+
+export const StyledContainer = styled.div`
+  ${(props) => {
+    const spaces = getSpaces(props);
+    return `
+      margin-top: ${spaces?.l}px;
+    `;
+  }}
+`;
+
+export const StyledExplainerTitle = styled.div`
+  ${fontHeaderXs}
+  display: flex;
+  align-items: center;
+  color: black;
+`;
+
+export const StyledFiltersSection = styled.div`
+  display: flex;
+  flex-direction: row;
+  ${(props) => {
+    const spaces = getSpaces(props);
+    return `
+      margin-top: ${spaces?.m}px;
+    `;
+  }}
+`;
+
+export const StyledFilterGroup = styled.div`
+  ${(props) => {
+    const spaces = getSpaces(props);
+    return `
+      margin-left: ${spaces?.xl}px;
+    `;
+  }}
+`;
+
+export const StyledFilterGroupName = styled.p`
+  ${fontHeaderXs}
+  color: black;
+`;
+
+export const StyledNewTabLink = styled(NewTabLink)`
+  ${(props) => {
+    const colors = getColors(props);
+
+    return `
+      &:hover {
+        color: ${colors?.primary[600]};
+        text-decoration: none;
+      }
+    `;
+  }}
+`;
+
+export const StyledInputDropdown = styled(InputDropdown)`
+  width: 200px;
+`;
+
+const DROPDOWN_WIDTH = "200px";
+
+// High specificity label styling is to beat out czifui coloring.
+// Might be possible to do by passing a `styled` InputDropdownComponent
+// to Dropdown, but might also not be. I think the czifui coloring we are
+// trying to beat here might be a bug though, so I think this is the better
+// way for now. I (Vince) will write up an issue April 14, 2022.
+export const StyledDropdown = styled(Dropdown)`
+  width: ${DROPDOWN_WIDTH};
+  .MuiButton-label > span {
+    color: black;
+  }
+` as typeof Dropdown; // assert b/c `styled` causes an interface hiccup;
+
+// DropdownPopper (for use with Dropdown's PopperComponent prop) needs to
+// have a `placement` prop to set where it anchors against Dropdown opener.
+const ConfiguredDropdownPopper = (props: any) => {
```

Comment:
I kind of feel like this doesn't belong in the style sheet. Why not include it in the index.tsx file?

Reply:
I waffled on this one. My thinking was that this configuration was ultimately visual. So since `style.tsx` is about visual stuff (in my mind), I figured it made the most sense in here rather than in `index.tsx` because it's not really about anything logic-process-wise. I also just liked passing a `StyledDropdownPopper` as the `PopperComponent` without needing to give `StyledDropdownPopper` any additional props.

If you think the `style.tsx` is more about CSS though rather than visual stuff generally, I can move it around.

Reply:
I do think the style files are stylesheets, and not visual stuff. I don't think component composition belongs in the style files, and I feel pretty strongly about that.

Reply:
That's reasonable, I'll keep with the stylesheet intention in the future. I moved the configuration over to the index.tsx and went back to a style.ts that's just a bunch of `styled` calls.